documentation/modules/managing/proc-cluster-recovery-volume.adoc

```diff
@@ -6,21 +6,13 @@
 = Recovering a deleted cluster from persistent volumes
 
 [role="_abstract"]
-This procedure describes how to recover a deleted cluster from persistent volumes (PVs).
+This procedure describes how to recover a deleted cluster from persistent volumes (PVs) by recreating the original `PersistentVolumeClaim` (PVC) resources.
 
-In this situation, the Topic Operator identifies that topics exist in Kafka, but the `KafkaTopic` resources do not exist.
+If the Topic Operator is deployed, you can also recover `KafkaTopic` resources.
+This must be done before the cluster is started so that the corresponding topics are not deleted.
 
-When you get to the step to recreate your cluster, you have two options:
-
-. Use _Option 1_ when you can recover all `KafkaTopic` resources.
-+
-The `KafkaTopic` resources must therefore be recovered before the cluster is started so that the corresponding topics are not deleted by the Topic Operator.
-
-. Use _Option 2_ when you are unable to recover all `KafkaTopic` resources.
-+
-In this case, you deploy your cluster without the Topic Operator, delete the Topic Operator topic store metadata, and then redeploy the Kafka cluster with the Topic Operator so it can recreate the `KafkaTopic` resources from the corresponding topics.
-
-NOTE: If the Topic Operator is not deployed, you only need to recover the `PersistentVolumeClaim` (PVC) resources.
+NOTE: The procedure does not include recovery of `KafkaUser` resources, which must be recreated manually.
+If passwords and certificates need to be retained, secrets must be recreated before creating the `KafkaUser` resources.
```

Comment:
It seems weird we include the topics but not include the users. But if we want to stick with it, we should explain that:
* If the UO is enabled
* And the KafkaUser resources not recreated
=> then the users will be deleted from the Kafka cluster immediately after the recovery.

Reply:
Considering the update to the note, maybe we're better including `KafkaUser` recovery too. I'll change the steps.